en/config_mc/filter_tuning.md

```diff
@@ -0,0 +1,98 @@
+- PX4 software internals: the sensor signals need to be read in the driver and then pass through the controller to the output driver.
+- The IO chip (MAIN pins) adds about 5.4 ms latency compared to using the AUX pins (this does not apply to a *Pixracer* or *Omnibus F4*, but does apply to a Pixhawk).
+  To avoid the IO delay, disable [SYS_USE_IO](../advanced_config/parameter_reference.md#SYS_USE_IO) and attach the motors to the AUX pins instead.
+- PWM output signal: enable One-Shot to reduce latency ([PWM_RATE](../advanced_config/parameter_reference.md#PWM_RATE)=0).
+
+Below we look at the impact of the low pass filters.
+
+## Filters
+
+This is the filtering pipeline for the controllers in PX4:
+- On-chip DLPF for the gyro sensor.
+  This is disabled on all chips where it can be disabled (if not, the cutoff frequency is set to 98Hz and it is sampled at 1kHz).
+- Low-pass filter on the gyro sensor data.
+  It can be configured with the [IMU_GYRO_CUTOFF](../advanced_config/parameter_reference.md#IMU_GYRO_CUTOFF) parameter.
+  :::note
+  Sampling and filtering is always performed at the full raw sensor rate (commonly 8Hz).
+  :::
+- A separate low-pass filter on the D-term.
+  The D-term is most susceptible to noise while slightly increased latency does not negatively affect performance.
+  For this reason the D-term has a separately-configurable low-pass filter, [IMU_DGYRO_CUTOFF](../advanced_config/parameter_reference.md#IMU_DGYRO_CUTOFF).
+- A slewrate filter on the motor outputs ([MOT_SLEW_MAX](../advanced_config/parameter_reference.md#MOT_SLEW_MAX)).
+  Generally not used.
+
+To reduce the control latency, we want to increase the cutoff frequency for the low-pass filters.
+The effect on latency of increasing `IMU_GYRO_CUTOFF` is approximated below.
+
+Cuttoff (Hz) | Delay approx. (ms)
+--- | ---
+30 | 8
+60 | 3.8
+120 | 1.9
+
+However this is a trade-off as increasing `IMU_GYRO_CUTOFF` will also increase the noise of the signal that is fed to the motors.
+Noise on the motors has the following consequences:
+- Motors and ESCs can get hot, to the point where they get damaged.
+- Reduced flight time because the motors continuously change their speed.
+- Visible random small twitches.
+
+The best filter settings depend on the vehicle.
+The defaults are set conservatively — such that they work on lower-quality setups as well.
+
+## Filter Tuning
+
+First make sure to have the high-rate logging profile activated ([SDLOG_PROFILE](../advanced_config/parameter_reference.md#SDLOG_PROFILE) parameter).
+[Flight Review](../getting_started/flight_reporting.md) will then show an FFT plot for the roll, pitch and yaw controls.
+
+:::warning
+Do not try to fix a vehicle that suffers from high vibrations with filter tuning!
+Instead:
+- Confirm that PID gains, in particular D, are not set too high as this can show up as vibrations.
+- Fix the vehicle hardware setup.
```

Comment:
I'd separate like this:
```suggestion
- Do not try to fix a vehicle that suffers from high vibrations with filter tuning!
Instead fix the vehicle hardware setup.
- Confirm that PID gains, in particular D, are not set too high as this can show up as vibrations.
```